libvirt/resource_libvirt_domain.go

```diff
@@ -28,6 +28,9 @@ type DomainMeta struct {
 // PoolSync exported pool sync
 var PoolSync = NewLVirtPoolSync()
 
+// destroyDomainRequestedByUser tracks the desired state of the domain. If set to true, the domain will be destroyed after its creation.
+var destroyDomainRequestedByUser bool
```

Comment:
Do we really need a global here? Isn't it a domain attribute?

Reply:
@inercia  yop this i will fix, is a mistake from my part.